src/jsifier.js

```diff
@@ -245,8 +245,10 @@ function JSify(data, functionsOnly) {
         if (!(finalName in IMPLEMENTED_FUNCTIONS)) {
           if (VERBOSE || ident.substr(0, 11) !== 'emscripten_') { // avoid warning on emscripten_* functions which are for internal usage anyhow
             if (!LINKABLE) {
-              if (ERROR_ON_UNDEFINED_SYMBOLS) error('unresolved symbol: ' + ident);
-              else if (VERBOSE || WARN_ON_UNDEFINED_SYMBOLS) warn('unresolved symbol: ' + ident);
+              if (ERROR_ON_UNDEFINED_SYMBOLS) {
+                error('undefined symbol: ' + ident);
+                warnOnce('To disable errors for undefined symbols use `-s ERROR_ON_UNDEFINED_SYMBOLS=1`')
```

Comment:
Nice error messaging, should make the migration simpler
... Although, that should probably read `-s ERROR_ON_UNDEFINED_SYMBOLS=0` instead of `=1`